index.js

```diff
@@ -25,20 +26,16 @@ input.addEventListener('keyup', function (event) {
 init();
 
 function init() {
-    const tasksString = localStorage.getItem('tasks');
-
-    if (!tasksString) {
-        localStorage.setItem(TASK_KEY, []);
-    } else {
-        tasks = JSON.parse(tasksString);
-        localStorage.setItem(TASK_KEY, JSON.stringify(tasks));
-        for (let i = 0; i < tasks.length; i++) {
-            renderTask(tasks[i]);
-            if (!tasks[i].completed) {
+    const tasksString = localStorage.getItem('tasks') || '[]';
+    tasks = JSON.parse(tasksString);
+    tasks
+        .forEach(function(currentTask) {
```

Comment:
Use arrow function for the callback functions (since you already used it in few places, so to be consistent)

Reply:
Line 16 as well